projects/questionset-editor-library/src/lib/components/question/question.component.spec.data.ts

```diff
@@ -151,22 +152,22 @@ export const mockData = {
               {
                 label: "<p>option 1</p>",
                 value: 0,
-                hints: {
-                  en: "test hint 1",
+                hint: {
```

Comment:
This is not as per QuML spec, please check and update the same. Here is the example for storing hints in interaction- 
`hint: "70c82bf5-9459-4c43-8897-0e58b7e1da64"`